README.md

```diff
@@ -174,12 +174,14 @@ For more details, see the [update system documentation](sources/updater/).
 
 ### Update methods
 
+#### Automated updates
+
 There are several ways of updating your Bottlerocket hosts.
+We provide tools for automatically updating hosts, as well as an API for direct control of updates.
```

Comment:
```suggestion
There are several ways of updating your Bottlerocket hosts.
We provide tools for automatically updating hosts, as well as an API for direct control of updates.

#### Automated updates
```
This text is introducing the distinction between automated and manual updates, so it should go before one of the specific headers.

Reply:
Done